deployment/cdk/app.py

```diff
@@ -359,6 +359,18 @@ def __init__(
             certificate=certificate
         )
 
+        if settings.additional_certificate_arn is not None and settings.additional_certificate_arn != '':
+            for listener in fargate_service.load_balancer.listeners:
+                if listener.to_string() != http_listener.to_string():
```

Comment:
This logic is pretty awful but there was no clean way I could find in CDK to access the HTTPS listener and add more certificates to it